app-shells/starship/starship-0.58.0.ebuild

```diff
@@ -0,0 +1,290 @@
+	pest_meta-2.1.3
+	pin-project-lite-0.2.7
+	pin-utils-0.1.0
+	pkg-config-0.3.19
+	polling-2.1.0
+	ppv-lite86-0.2.10
+	predicates-1.0.8
+	predicates-core-1.0.2
+	predicates-tree-1.0.3
+	proc-macro-crate-0.1.5
+	proc-macro-crate-1.1.0
+	proc-macro-hack-0.5.19
+	proc-macro-nested-0.1.7
+	proc-macro2-1.0.29
+	process_control-3.1.0
+	quick-xml-0.22.0
+	quote-0.3.15
+	quote-1.0.9
+	rand-0.8.4
+	rand_chacha-0.3.1
+	rand_core-0.6.3
+	rand_hc-0.3.1
+	rayon-1.5.1
+	rayon-core-1.9.1
+	redox_syscall-0.1.57
+	redox_syscall-0.2.10
+	redox_users-0.3.5
+	redox_users-0.4.0
+	regex-1.5.4
+	regex-syntax-0.6.25
+	remove_dir_all-0.5.3
+	rust-argon2-0.8.3
+	rust-ini-0.17.0
+	ryu-1.0.5
+	schannel-0.1.19
+	scoped-tls-1.0.0
+	scopeguard-1.1.0
+	security-framework-2.3.1
+	security-framework-sys-2.4.2
+	semver-1.0.4
+	serde-1.0.130
+	serde_derive-1.0.130
+	serde_json-1.0.68
+	serde_repr-0.1.7
+	serde_urlencoded-0.6.1
+	sha-1-0.8.2
+	shadow-rs-0.7.1
+	shell-words-1.0.0
+	slab-0.4.4
+	socket2-0.4.2
+	starship-0.58.0
+	starship_module_config_derive-0.2.1
+	static_assertions-1.1.0
+	strsim-0.8.0
+	strsim-0.10.0
+	strum-0.8.0
+	strum_macros-0.8.0
+	syn-0.11.11
+	syn-1.0.76
+	synom-0.11.3
+	sys-info-0.9.0
+	tempfile-3.2.0
+	term_size-0.3.2
+	textwrap-0.11.0
+	thiserror-1.0.29
+	thiserror-impl-1.0.29
+	time-0.1.44
+	tinyvec-1.4.0
+	tinyvec_macros-0.1.0
+	toml-0.5.8
+	treeline-0.1.0
+	typenum-1.14.0
+	ucd-trie-0.1.3
+	unicode-bidi-0.3.6
+	unicode-normalization-0.1.19
+	unicode-segmentation-1.8.0
+	unicode-width-0.1.9
+	unicode-xid-0.0.4
+	unicode-xid-0.2.2
+	uom-0.30.0
+	url-2.2.2
+	urlencoding-2.1.0
+	utf8-width-0.1.5
+	vcpkg-0.2.15
+	vec_map-0.8.2
+	version_check-0.9.3
+	versions-3.0.3
+	void-1.0.2
+	waker-fn-1.1.0
+	wasi-0.9.0+wasi-snapshot-preview1
+	wasi-0.10.0+wasi-snapshot-preview1
+	wepoll-ffi-0.1.2
+	which-4.2.2
+	wildmatch-1.1.0
+	winapi-0.3.9
+	winapi-i686-pc-windows-gnu-0.4.0
+	winapi-x86_64-pc-windows-gnu-0.4.0
+	winrt-0.4.0
+	winrt-notification-0.2.4
+	xml-rs-0.6.1
+	yaml-rust-0.4.5
+	zbus-1.9.1
+	zbus_macros-1.9.1
+	zvariant-2.8.0
+	zvariant_derive-2.8.0
+"
+
+inherit cargo
+
+DESCRIPTION="The minimal, blazing-fast, and infinitely customizable prompt for any shell!"
+HOMEPAGE="https://github.com/starship/starship"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+
+LICENSE="
+	|| ( Apache-2.0 Apache-2.0-with-LLVM-exceptions MIT )
+	|| ( Apache-2.0 Boost-1.0 )
+	|| ( Apache-2.0 MIT )
+	|| ( Apache-2.0 MIT ZLIB )
+	|| ( MIT Unlicense )
+	Apache-2.0
+	BSD-2
+	ISC
+	MIT
+	MPL-2.0
+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+battery +http tls-vendored"
+
+DEPEND="
+	>=dev-libs/libgit2-1.2.0:=
+	!tls-vendored? ( dev-libs/openssl:0= )
+"
+RDEPEND="${DEPEND}
+"
+
+# Rust packages ignore CFLAGS and LDFLAGS so let's silence the QA warnings
+QA_FLAGS_IGNORED="usr/bin/starship"
+
+src_configure() {
+	myfeatures=(
+		$(usev battery)
+		$(usev http)
+		$(usev tls-vendored)
+	)
+	cargo_src_configure --no-default-features
+}
+
+src_install() {
+	cargo_src_install
+
+	local DOCS=( README.md )
```

Comment:
By default einstalldocs will install `README*` among others, so it can pickup this README.md this without needing this line.

Unless you mean to stop it from picking up other files.